src/blackduck-security-action/bridge-cli.ts

```diff
@@ -32,14 +32,27 @@ export class Bridge {
     this.bridgeExecutablePath = ''
     this.bridgePath = ''
     this.bridgeArtifactoryURL = constants.BRIDGE_CLI_ARTIFACTORY_URL
-    this.bridgeUrlPattern = this.bridgeArtifactoryURL.concat('$version/bridge-cli-$version-$platform.zip')
-    this.bridgeUrlLatestPattern = this.bridgeArtifactoryURL.concat('latest/bridge-cli-$platform.zip')
+    this.bridgeUrlPattern = this.bridgeArtifactoryURL.concat('$version/bridge-cli-bundle-$version-$platform.zip')
+    this.bridgeUrlLatestPattern = this.bridgeArtifactoryURL.concat('latest/bridge-cli-bundle-$platform.zip')
   }
 
   private getBridgeDefaultPath(): string {
     let bridgeDefaultPath = ''
+    const subFolder = 'bridge-cli-bundle-'.concat(this.getOSPlatform())
     const osName = process.platform
+    if (osName === 'darwin') {
+      bridgeDefaultPath = path.join(process.env['HOME'] as string, BRIDGE_CLI_DEFAULT_PATH_MAC, subFolder)
+    } else if (osName === 'linux') {
+      bridgeDefaultPath = path.join(process.env['HOME'] as string, BRIDGE_CLI_DEFAULT_PATH_LINUX, subFolder)
+    } else if (osName === 'win32') {
+      bridgeDefaultPath = path.join(process.env['USERPROFILE'] as string, BRIDGE_CLI_DEFAULT_PATH_WINDOWS, subFolder)
+    }
 
+    return bridgeDefaultPath
+  }
+  private getBridgeDownloadDefaultPath(): string {
```

Comment:
can we call getBridgeZCliDefaultDownloadPath instead of  getBridgeDownloadDefaultPath?

Reply:
made the changes please check